internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -103,13 +103,6 @@
           ]
         },
         {
-          "name": "nameHashing",
```

Comment:
It is my understanding that deprecation cycles are useful when evolving APIs, in minor versions. Zinc 1.0 brings to the table a major version with a completely reworked API. IMO, we should not use them here.